src/angular-public/processflow/processflow/processflow.component.ts

```diff
@@ -21,8 +20,4 @@ export class ProcessflowComponent extends ProcessflowBase<ProcessflowStepCompone
   /** @docs-private */
   @ContentChildren(ProcessflowStepComponent)
   steps: QueryList<ProcessflowStepComponent>;
-
-  constructor(changeDetectorRef: ChangeDetectorRef) {
```

Comment:
does this really work? ;-)

Reply:
Should now work with the `@Directive()` decorator on the base class.